### PR TITLE
App ID updates in project list and info page after clicking save

### PR DESCRIPTION
**Pull request by kflores772**

### CODE REVIEW

**Comment by kflores772** [comment]
This fixes #84. @rcruz please test

File: lib/ui/views/projectInfo.js [hunk 1]
```diff
@@ -204,6 +204,8 @@ ProjectInfoView = Backbone.View.extend({
                 model.set("projectId", model.get("config").widget.id);
                 model.sync("update");
             }
+            window.views.projectList.render();
+            view.render();
```
Comment (rcruz) [inline lib/ui/views/projectInfo.js:208]: Might not need this. It will reload the main content area and might scroll you back up to the top.

**Comment by kflores772** [comment]
view.render will reload and scroll back up to the top of the page. However if not included, the main title on the project info page will not update with the new App ID

**Comment by rcruz** [comment]
r+, tested and working

**Pull request closed without merging**